src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -36,20 +36,33 @@ public enum TransformerScope
         Everything = Training | Testing | Scoring
     }
 
+    [BestFriend]
+    internal interface ITransformerAccessor
```

Comment:
>ITransformerAccessor [](start = 23, length = 20)

As we discussed earlier this is the sort of thing that really bears some explanation.